### PR TITLE
added CONTRIBUTION.md

### LINKED ISSUE #2: Add CONTRIBUTION.md
This project is Open Source so having a decent Contribution guide will be great. No specific requirements, just a basic CONTRIBUTION.md 📃

### PR DESCRIPTION
**Pull request by danger-ahead**
Addresses #2

### CODE REVIEW

**Comment by danger-ahead** [comment]
@mindninjaX What's the required change? You haven't mentioned.

File: CONTRIBUTIONS.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+```$ git push origin <feature-branch>```
+    
+Now you are ready to help the project by opening a pull request means you now tell the project managers to add the feature or bugfix to original repository. You can open a pull request by clicking on green icon -
+
+<p align="center">  <img  src="https://i.imgur.com/aGaqAD5.png">  </p>
+
+Remember your upstream base branch should be master and source should be your feature branch. Click on create pull request and add a name to your pull request. You can also describe your feature.
+
+Awesome! You have made your first contribution. If you have any doubts please let me know in the comments.
```
Comment (mindninjaX) [inline CONTRIBUTIONS.md]: @danger-ahead Just a small fix! 
Please change this line to:
```suggestion
Awesome! You have made your first contribution. If you have any doubts please let us know in the comments.
```

In a nutshell, all you have to do is change `me` to `us`.

**Review by mindninjaX — CHANGES_REQUESTED** [review]

**Comment by danger-ahead** [comment]
@mindninjaX done!

**Review by mindninjaX — APPROVED** [review]

**Comment by mindninjaX** [comment]
Hey @danger-ahead ! Everything looks great! Thanks, you so much for your contribution 😄 
Merging your PR ⭐

**Comment by mindninjaX** [comment]
Hey @danger-ahead! One last thing before we wrap up. I would highly recommend you create a separate branch before for all your next contributions. Directly making changes to the main branch is not a good open-source practice.

But anyways, thank you for your time & contribution 😄

**Comment by danger-ahead** [comment]
> Hey @danger-ahead! One last thing before we wrap up. I would highly recommend you create a separate branch before for all your next contributions. Directly making changes to the main branch is not a good open-source practice.
> 
> But anyways, thank you for your time & contribution smile

@mindninjaX will keep that in mind. thanks!

**Pull request merged**